src/app/portlets/content-types/form/content-types-form.component.html

```diff
@@ -22,11 +22,11 @@
     <div class="form__group">
         <div class="ui-g-6">
             <label for="content-type-form-publish-date-field" class="form__label">{{ dotMessageService.get('contenttypes.form.label.publish.date.field') }}</label>
-            <p-dropdown [options]="dateVarOptions" id="content-type-form-publish-date-field" name="publishDateVar" formControlName="publishDateVar" (onChange)="handleDateVarChange($event, 'publishDateVar')" [style]="{'min-width':'90px'}" tabindex="5"></p-dropdown>
+            <p-dropdown appendTo="body" [options]="dateVarOptions" id="content-type-form-publish-date-field" name="publishDateVar" formControlName="publishDateVar" (onChange)="handleDateVarChange($event, 'publishDateVar')" [style]="{'min-width':'90px'}" tabindex="5"></p-dropdown>
```

Comment:
When you use a dropdown or any overlay related component inside the dialog make sure you do `appendTo="body"` because if not `overflow: scroll` of the content of the dialog will "hide" the overlay.